src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -441,7 +441,7 @@ private View InternalActionContent
         }
 
         /// <summary>
-        /// AccessibilityGetName.
+        /// Gets AccessibilityName.
```

Comment:
It does not get `AccessibilityName`, because `AccessibilityName` is empty (and `AccessibilityName` takes priority over `AccessibilityGetName` anyway); it gets the `Title` (or `"accessibility name"` as described in `DefaultLinearItem`).

Reply:
Got it!